<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="pensionboxdewesh-VivoBook-ASUSLaptop-X415JAB-X415JA" failures="1" tests="2" name="Test" time="29.124" errors="0" timestamp="2024-03-13T20:34:30 IST">
  <testcase classname="Assertion.CreateContactaWithLeadsourceAssertionTest" name="createContactWithLeadSource" time="6.191"/>
  <testcase classname="Assertion.CreateNewContactaWithAssertionTest" name="CreateNewContactsWithMandatoryFieldTest" time="4.375">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at Assertion.CreateNewContactaWithAssertionTest.CreateNewContactsWithMandatoryFieldTest(CreateNewContactaWithAssertionTest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- CreateNewContactsWithMandatoryFieldTest -->
  <testcase classname="genericUtility.BaseClass" name="@BeforeSuite bsConfig" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite bsConfig -->
  <testcase classname="genericUtility.BaseClass" name="@AfterSuite asConfig" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite asConfig -->
</testsuite> <!-- Test -->
